src/AttachVS/AttachVs.cs

```diff
@@ -38,20 +36,25 @@ internal static bool AttachVSToProcess(int? pid, int? vsPid)
             else
             {
                 Trace($"Parent VS not found, finding the first VS that started.");
-                var processes = Process.GetProcesses().Where(p => p.ProcessName == "devenv").Select(p =>
-                {
-                    try
+                var processes = Process.GetProcesses()
+                    .Where(p => p.ProcessName == "devenv")
+                    .Select(p =>
                     {
-                        return new { Process = p, p.StartTime, p.HasExited };
-                    }
-                    catch
-                    {
-                        return null;
-                    }
-                }).Where(p => p != null && !p.HasExited).OrderBy(p => p.StartTime).ToList();
+                        try
+                        {
+                            return new { Process = p, p.StartTime, p.HasExited };
+                        }
+                        catch
+                        {
+                            return null;
+                        }
+                    })
+                    .Where(p => p != null && !p.HasExited)
+                    .OrderBy(p => p!.StartTime)
+                    .ToList();
 
-                var firstVs = processes.FirstOrDefault();
-                Trace($"Found VS {firstVs.Process.Id}");
+                var firstVs = processes.First();
```

Comment:
We were using `FirstOrDefault` but then doing member access. So it's unclear to me whether or not we expect to have an empty list.

Reply:
I will have to look at the logic again. Plan was originally to: pickup the "current" process from env variable, or use the current. Then for that  pickup parent VS, and if not found fallback to first VS started. And if not found then fail.